x-pack/plugins/actions/server/lib/create_action_event_log_record_object.test.ts

```diff
@@ -345,6 +356,7 @@ describe('createActionEventLogRecordObject', () => {
         ],
         action: {
           name: 'test name',
+          type_id: '.slack',
```

Comment:
this must have been the source of the question on snake- vs camel- case hahahaha

This `type_id` seems out of place to me.  Is this the way we do it with rule id's?

Reply:
No it was SO name :) Like, `action` or `alert`.
This is a filed, it is the same for the rule, just the name is rule_type_id... but `rule.rule_type_id looks` weird to me...